tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -24,7 +24,3 @@ Other free text and markdown formatting can be used elsewhere in the document if
 ### [empty subfolder inside a folder to be uploaded is not created on the server](https://github.com/owncloud/web/issues/6348)
 
 - [webUIUpload/upload.feature:43](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIUpload/upload.feature#L43)
-
-### [PROPFIND to sub-folder of a shared resources with same name gives 404](https://github.com/owncloud/ocis/issues/3859)
```

Comment:
what is the status of this issue? I see it is still open

Reply:
The mentioned issue doesn't exist on the latest oCIS master. The expected behavior `(1)` to be added on the second share with same name, and resources can be accessed via propfind. So, that issue is closed now.

Reply:
Then please, find the API test coverage for that one and if there is none we have to add one